### PR TITLE
Add Lizmap WKT check method

### PR DESCRIPTION
**Pull request by rldhont**
Before parsing WKT string, checking it.

* Funded by 3liz

### CODE REVIEW

File: lizmap/modules/lizmap/classes/lizmapWkt.class.php [hunk 1]
```diff
@@ -18,6 +18,20 @@ class lizmapWkt
...
     );
 
+    /**
+     * Check if the WKT string is well formed.
+     *
+     * @param string $wkt A WKT string
+     *
+     * @return bool The WKT string is well formed
+     */
+    public static function check($wkt)
```
Comment (mdouchin) [inline lizmap/modules/lizmap/classes/lizmapWkt.class.php:28]: Please use this new function in the following `public static function parse($wkt)`

**Review by mdouchin — CHANGES_REQUESTED** [review]
LGTM, just a minor refactoring required

File: lizmap/modules/lizmap/classes/lizmapWkt.class.php (hunk 1 above)
Comment (rldhont) [reply lizmap/modules/lizmap/classes/lizmapWkt.class.php:28]: check returns `bool` not the `$matches` needed in `parse`.

**Comment by rldhont** [comment]
Waiting for https://github.com/3liz/lizmap-web-client/pull/2956 before merging.

File: lizmap/modules/lizmap/classes/lizmapWkt.class.php (hunk 1 above)
Comment (rldhont) [reply lizmap/modules/lizmap/classes/lizmapWkt.class.php:29]: @mdouchin `check` is reused in `parse`.

**Comment by rldhont** [comment]
@mdouchin I have also added unit tests.

**Review by mdouchin — APPROVED** [review]
Thanks

**Pull request merged**